docs/workflows/lcmsworkflow/lcms-workflow.md

```diff
@@ -1,25 +1,30 @@
 # LC-MS Workflow
-## Raw data processing
+The following workflow is intended to turn raw LC-MS data into a table of all the detected features and corrisponding signal intensity detected in each sample. A schematic representation of the workflow is shown below:
 
-### Raw data import
+![workflow-image](https://github.com/titodamiani/mzmine_documentation/blob/e90deb3ccb11b18c94f7f27ed76f71c818640091/docs/workflows/lcmsworkflow/workflow-image.png)
```

Comment:
use relative paths for images please. it would be jsut ![workflow-image](workflow-image.png) here i think